env/feb20/sim_python_no-threads.yaml

```diff
@@ -0,0 +1,21 @@
+spack:
+  definitions:
+    - when: platform == 'linux'
+      root: [root@6.18.04+fortran+gdml+http+memstat+pythia6+pythia8+vc~vdt+python+tmva]
+    - when: platform == 'darwin'
+      root: [root@6.18.04+fortran+gdml+http+memstat+pythia6+pythia8+vc~vdt+python+tmva+aqua]
+  specs:
+    #    - cmake@3.13.4
+    - googletest@1.10.0
+    - boost@1.70.0
+    - fairlogger@1.6.1
```

Comment:
> @dennisklein which versions of FairMQ & co will go into feb20 release? Can you add corresponding entries to the environment?

@kresan @MohammadAlTurany I would pin FairMQ, and DDS for jun19, because it was already defined in the legacy code. For new - spack-only - releases @ChristianTackeGSI and I remember both when we (all four of us) last had discussed this, we reached the conclusion to only pin versions of physics-sensitive packages, didn't we?

If you still agree with what I remember, I would suggest to remove the lines above:
```diff
@@ 8,4 - 8,0 @@
-    #    - cmake@3.13.4
-    - googletest@1.10.0
-    - boost@1.70.0
-    - fairlogger@1.6.1
```

Or did I miss an argument why we changed our opinions on this?

Reply:
@dennisklein sorry, since our conclusion was not noted on the paper, I just forgot it. Agree, I will remove the lines you mentioned from the environment feb20.